requirements.txt

```diff
@@ -9,7 +9,8 @@ cryptography==0.9.1
 debtcollector==0.5.0
 docker-py==1.3.1
 effect==0.1a13
-eliot==0.8.0
+eliot==0.9.0
+eliot-tree==15.3.0
```

Comment:
Docs can come later, probably.  When folks show up with a question it will be nice to be able to tell them to use eliot-tree and have it already be on their system.